aa-sdk/core/src/actions/smartAccount/internal/waitForUserOperation.ts

```diff
@@ -0,0 +1,56 @@
+import type { Chain, Client, Hex, Transport } from "viem";
```

Comment:
⚠️ **[eslint]** <[@typescript-eslint/no-unused-vars](https://typescript-eslint.io/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'Hex' is defined but never used.
<!-- __reviewdog__:ChA3Nzg5NWQzMWVhNjM5YWRkEgZyZGpzb24= -->